docs/src/main/sphinx/object-storage/file-system-gcs.md

```diff
@@ -75,10 +75,12 @@ Cloud Storage:
   - Authentication type to use for Google Cloud Storage access. Default to `SERVICE_ACCOUNT`.
   Supported values are:
     * `SERVICE_ACCOUNT`: loads credentials from the environment. Either `gcs.json-key` or
-      `gcs.json-key-file-path` can be set in addition to override the default
-      credentials provider.
+      `gcs.json-key-file-path` can be set.
     * `ACCESS_TOKEN`: usage of client-provided OAuth 2.0 token to access Google
       Cloud Storage.
+    * `NO_CREDENTIALS`: allows accessing Google Cloud Storage without providing
```

Comment:
This is misleading. "No credentials" would seem to imply "unauthenticated", but that is not the case based on the description.

It's also somewhat confusing that both SERVICE_ACCOUNT and NO_CREDENTIALS "acquire application default credentials from the environment". So, what's the difference?